tsconfig.json

```diff
@@ -1,25 +1,17 @@
 {
```

Comment:
I haven't checked this is backward-compatible yet, but running `npm pack` with both old and new settings and verifying the diff (or any technique like that) would be worth doing (you might have already done this tho!)

Reply:
I did add example integration tests in the test PR (see [here](https://github.com/slackapi/bolt-js/pull/2259/checks)); the application TypeScript seems to compile fine when integrated with this PR.

I will do a manual testing pass with the other well-known sample bolt-js apps we have to validate everything works as desired. I don't think the emitted JavaScript should be different, since it is still set to emit `commonjs` (just like before) and module resolution is still `node` (just like before), but I will validate manually. And if there are any other sample applications we should integrate with, let me know and I can add it to the CI (at least compilation-level validation with TypeScript projects is an easy integration test to run).

Reply:
Tested works fine with the following samples:

- https://github.com/slack-samples/bolt-js-custom-step-template
- https://github.com/slack-samples/bolt-js-starter-template
- https://github.com/slack-samples/bolt-ts-custom-function-template
- https://github.com/slack-samples/bolt-ts-starter-template

Reply:
Thanks for double-checking this! 🙇